crates/iota-graphql-client/examples/transactions_with_function.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
```

Comment:
could use the `Result` from the error module?

Reply:
https://github.com/iotaledger/iota-rust-sdk/pull/195/commits/fd5445221e60d0e5cd2ae48ed20146585c5571de